Dockerfile

```diff
@@ -1,18 +1,21 @@
 # This file is part of REANA.
-# Copyright (C) 2021 CERN.
+# Copyright (C) 2021, 2022 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 # Install base image and its dependencies
-FROM python:3.8-slim
```

Comment:
FWIW it was necessary to abandon "slim" in order to compile XRootD, so the images are now bigger:

```
reanahub/reana-workflow-engine-snakemake                                            latest                  621dedadaab6   2 hours ago     1.24GB
reanahub/reana-workflow-engine-snakemake                                            0.9.0-alpha.3           47ed8fb3ce7f   3 weeks ago     499MB
```

The usual Snakemake demo test suite is working fine.

Reply:
... as is an XRootD motivated demo with the following Snakefile:

```
from snakemake.remote.XRootD import RemoteProvider as XRootDRemoteProvider

XRootD = XRootDRemoteProvider(stay_on_remote=True)
file_numbers = XRootD.glob_wildcards("root://eospublic.cern.ch//eos/opendata/lhcb/MasterclassDatasets/D0lifetime/2014/mclasseventv2_D0_{n}.root").n

rule all:
    input:
        expand("local_data/mclasseventv2_D0_{n}.root", n=file_numbers)

rule make_data:
    input:
        XRootD.remote("root://eospublic.cern.ch//eos/opendata/lhcb/MasterclassDatasets/D0lifetime/2014/mclasseventv2_D0_{n}.root")
    output:
        'local_data/mclasseventv2_D0_{n}.root'
    container:
        "docker://opensciencegrid/osgvo-el7:latest"
    shell:
        'xrdcp {input[0]} {output[0]}'
```

Downloading files well:

![snakemake-xrootd](https://user-images.githubusercontent.com/517546/165535881-2e840b4a-fc17-4941-84f5-c973047164c6.png)